src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -18,10 +21,174 @@ private static void Main(string[] args)
+                        }
+                        return reason;
+                    }
+                }
+                catch (Exception ex)
+                {
+                    return ex.Message;
+                }
+            }
+
+            return null;
+        }
+
+        private static void StopBridgeIfLocal(CommandLineArguments commandLineArgs)
+        {
+            if (IsBridgeHostLocal(commandLineArgs.BridgeConfiguration))
+            {
+                StopBridge(commandLineArgs);
+            }
+            else
+            {
+                Console.WriteLine("The Bridge on host {0} is not running locally and will not be stopped.",
+                                    commandLineArgs.BridgeConfiguration.BridgeHost);
+                Console.WriteLine("Use 'Bridge.exe /stop' to stop a Bridge on another machine.");
+            }
+        }
+
+        private static void StopBridge(CommandLineArguments commandLineArgs)
+        {
+            string pingResult = PingBridge(commandLineArgs.BridgeConfiguration.BridgeHost,
+                                           commandLineArgs.BridgeConfiguration.BridgePort);
+            if (pingResult != null)
+            {
+                Console.WriteLine("The Bridge is not running.");
+                Environment.Exit(0);
+            }
+
+            string bridgeUrl = String.Format("http://{0}:{1}/Bridge", commandLineArgs.BridgeConfiguration.BridgeHost, commandLineArgs.BridgeConfiguration.BridgePort);
+            string problem = null;
+
+            // We stop the Bridge using a DELETE request.
+            // If the Bridge is running on localhost, it will be running
+            // in a different process on this machine.
+            using (HttpClient httpClient = new HttpClient())
+            {
+                Console.WriteLine("Stopping Bridge at {0}", bridgeUrl);
+                try
+                {
+                    var response = httpClient.DeleteAsync(bridgeUrl).GetAwaiter().GetResult();
+                    if (!response.IsSuccessStatusCode)
+                    {
+                        problem = String.Format("{0}Bridge returned unexpected status code='{1}', reason='{2}'",
+                                                    Environment.NewLine, response.StatusCode, response.ReasonPhrase);
+                        if (response.Content != null)
+                        {
+                            string contentAsString = response.Content.ReadAsStringAsync().GetAwaiter().GetResult();
+                            if (contentAsString.Length > 1000)
+                            {
+                                contentAsString = contentAsString.Substring(0, 999) + "...";
+                            }
+                            problem = String.Format("{0}, content:{1}{2}",
+                                                    problem, Environment.NewLine, contentAsString);
+                        }
+                        Console.WriteLine(problem);
```

Comment:
Same thing here with statuscode and reason not printing to console out if there is response content.